python/lsst/ts/m2/mock_command.py

```diff
@@ -23,72 +23,163 @@
 
 from lsst.ts.idl.enums import MTM2
 
-from . import DetailedState
+from . import DetailedState, CommandStatus
 
 
 __all__ = ["MockCommand"]
 
 
 class MockCommand:
-    """Mock command to simulate the execution of command in real hardware.
+    """Mock command to simulate the execution of command in real hardware."""
 
-    TODO (DM-30851):
-    Most of code will be implemented in DM-30851.
-
-    Attributes
-    ----------
-    system_enabled : `bool`
-        System is in the Enabled state or not.
-    """
-
-    def __init__(self):
-
-        # System is in the Enabled state or not
-        # TODO: This attribute will be removed in the work of DM-30851
-        self.system_enabled = False
-
-    async def enable(self, message, message_event):
+    async def enable(self, message, model, message_event):
         """Enable the system.
 
         Parameters
         ----------
         message : `dict`
             Command message.
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
         message_event : `MockMessageEvent`
             Instance of MockMessageEvent to write the event.
+
+        Returns
+        -------
+        model : `MockModel`
+            Mock model to simulate the M2 hardware behavior.
+        `CommandStatus`
+            Status of command execution.
         """
 
-        await message_event.write_force_balance_system_status(True)
+        model.actuator_power_on = True
+        command_success = model.switch_force_balance_system(True)
+
+        if not command_success:
+            model.actuator_power_on = False
+
+        await message_event.write_force_balance_system_status(
+            model.force_balance_system_status
+        )
 
         # Simulate the real hardware behavior
         await asyncio.sleep(5)
 
-        self.system_enabled = True
+        return model, CommandStatus.Success if command_success else CommandStatus.Fail
```

Comment:
Just a quick comment that in python it may be a bit dangerous to do 

```
if some_variable
```

it may give you some undesirable behavior if `some_variable` is not a boolean. I'd recommend you do:

```
CommandStatus.Success if command_success == True else CommandStatus.Fail
```

Reply:
Done in the commit: e99800d. Thanks!